src/components/checkbox-drawer/checkbox-drawer.js

```diff
@@ -71,7 +70,7 @@ class CheckboxDrawer extends LocalizeLabsElement(SkeletonMixin(LitElement)) {
 				<div class="d2l-input-checkbox-description d2l-skeletize">${this.description}</div>
 			</d2l-input-checkbox-spacer>
 			<d2l-input-checkbox-spacer class="d2l-input-checkbox-spacer">
-				<d2l-expand-collapse-content @d2l-expand-collapse-content-expand="${this._onExpandCollapseContentToggled}" @d2l-expand-collapse-content-collapse="${this._onExpandCollapseContentToggled}" ?expanded="${this.checked}">
+				<d2l-expand-collapse-content ?expanded="${this.checked}" ?aria-busy=${this._expandCollapseBusy} @d2l-expand-collapse-content-expand="${this._onExpandCollapseContentToggled}" @d2l-expand-collapse-content-collapse="${this._onExpandCollapseContentToggled}">
```

Comment:
This looks good, except `aria-busy` is oddly a `String`, so this should be:
```suggestion
				<d2l-expand-collapse-content ?expanded="${this.checked}" aria-busy=${this._expandCollapseBusy ? 'true' : 'false'} @d2l-expand-collapse-content-expand="${this._onExpandCollapseContentToggled}" @d2l-expand-collapse-content-collapse="${this._onExpandCollapseContentToggled}">
```

Reply:
Oh boy that's not confusing at all 😕 thanks for the heads up! I've changed that (I'll have to make sure the other places I've seen it in Discover are correct 😅 )